scripts/sanity_chk/sanitylib.py

```diff
@@ -2802,10 +2805,13 @@ def discard_report(self, filename):
                            "reason": reason}
                 cw.writerow(rowdict)
 
-    def target_report(self, outdir, append=False):
+    def target_report(self, outdir, suffix, append=False):
         platforms = {inst.platform.name for _, inst in self.instances.items()}
         for platform in platforms:
-            filename = os.path.join(outdir,"{}.xml".format(platform))
+            if suffix:
```

Comment:
you could set `suffix` to "" if it's `None` with a ternary and avoid the if/else.